### PR TITLE
add version number for search output

### PR DESCRIPTION
**Pull request by allyraza**
I tried implementing this, @ericmj  what do you think? 
https://github.com/hexpm/hex/issues/158

### CODE REVIEW

**Comment by ericmj** [comment]
Only the latest version should be shown. If you want to add an example config with version requirement it should be to `mix hex.info`.

**Comment by allyraza** [comment]
as you can see I am doing List.last just get the latest version number from the list

**Comment by ericmj** [comment]
Yes, but you are showing it as a dependency definition `{:foo, "~> 1.2.3"}`, just show it as `foo 1.2.3` instead. We also need to align the package name and version columns.

**Comment by allyraza** [comment]
the reason I added it as a tuple you can copy and paste direct from the terminal should i change it to `mix hex.info`

**Comment by ericmj** [comment]
Yes, they are separate features.

**Comment by allyraza** [comment]
will change it to that

**Comment by allyraza** [comment]
what do you think about the alignment we need to have a default width on the columns as the name length can vary

**Comment by ericmj** [comment]
The longest name needs to be found and spaces should be inserted on the shorter names to align it.

**Comment by allyraza** [comment]
I figured that would be good solution so I implemented it already please just take look

File: lib/mix/tasks/hex/search.ex [hunk 1]
```diff
@@ -17,8 +17,14 @@ defmodule Mix.Tasks.Hex.Search do
       [package] ->
         Hex.Utils.ensure_registry!()
 
-        Enum.each(Hex.Registry.search(package), fn pkg ->
-          Hex.Shell.info(pkg)
+        packages = Hex.Registry.search(package)
+        pkg_max_length = Enum.max_by(packages, fn p -> byte_size(p) end) |>
```
Comment (lexmag) [inline lib/mix/tasks/hex/search.ex]: `Enum.max_by(packages, &byte_size/1) |> byte_size`?

Comment (allyraza) [inline lib/mix/tasks/hex/search.ex]: that should also work thanks

**Comment by ericmj** [comment]
This is perfect, thank you!

**Pull request merged**